lib/dest/writeContents/utils.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var fs = require('graceful-fs');
+
+function utimes(writePath, stat, cb) {
+  if (stat.mtime) {
+    stat.atime = stat.atime || new Date();
```

Comment:
yeah, but we don't want to mutate the underlying vinyl object, which is what this is doing.  If you instead do `var atime = stat.atime || new Date()`, you won't be mutating the underlying object.  This needs a test too